purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Tecnativa - Manuel Calero
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests.common import TransactionCase
+from odoo import fields
+
+
+class TestPurchaseOrderUninvoiceAmount(TransactionCase):
+
+    def setUp(self):
+        super(TestPurchaseOrderUninvoiceAmount, self).setUp()
+        # environmet
+        self.purchase_order_model = self.env['purchase.order']
+        self.purchase_order_line_model = self.env['purchase.order.line']
+        self.account_invoice_model = self.env['account.invoice']
+        # company
+        self.company = self.env.ref('base.main_company')
+        # Partner
+        self.partner = self.env.ref('base.res_partner_1')
+        # Products
+        self.product_1 = self.env.ref('product.product_product_1')
```

Comment:
Avoid using demo data for tests, as it's prone to unexpected errors and interactions with third modules.

Reply:
Done